### PR TITLE
Hy-Be_Calendar

### PR DESCRIPTION
**Pull request by HristoYonkov**
1. Request for bringing dates for given month added
2. Categories changed to Bulgarian

### CODE REVIEW

File: src/controllers/eventController.js [hunk 1]
```diff
@@ -143,6 +144,43 @@ eventController.post('/like/:id', async (req, res) => {
...
+        res.status(400).json(error.message);
+        res.end();
+    }
+});
+
 module.exports = {
     eventController,
 };
+
+// TODO: Add it in utils later!
```
Comment (Getsov) [inline src/controllers/eventController.js:169]: Will we need this getMonthRange ? It’s even from 0 to 11

File: src/services/eventService.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ async function registerEvent(requestBody, requesterId) {
         longTitle: requestBody.longTitle,
         shortDescription: requestBody.shortDescription,
         longDescription: requestBody.longDescription,
-        dates: requestBody.dates.sort((a, b) =>  new Date(a.date) - new Date(b.date)),
+        dates: requestBody.dates.sort((a, b) => new Date(a.date) - new Date(b.date)),
```
Comment (Getsov) [inline src/services/eventService.js:13]: What if many dates?

File: src/services/eventService.js [hunk 3]
```diff
@@ -26,6 +26,7 @@ async function registerEvent(requestBody, requesterId) {
 
 async function findEventByID(eventId) {
     // TODO: make more tests with different values!
+    // TODO: Check if even is deleted?
```
Comment (Getsov) [inline src/services/eventService.js:29]: Even? May be event? It’s mandatory to check if event is deleted

File: src/services/eventService.js [hunk 4]
```diff
@@ -92,7 +93,7 @@ async function updateEvent(requestBody, existingEvent, isAdmin) {
             key !== 'likes' &&
             key !== 'isDeleted'
         ) {
-            key === 'dates' ? requestBody[key].sort((a, b) =>  new Date(a.date) - new Date(b.date)) : null;
+            key === 'dates' ? requestBody[key].sort((a, b) => new Date(a.date) - new Date(b.date)) : null;
```
Comment (Getsov) [inline src/services/eventService.js:96]: What if many dates?

File: src/controllers/eventController.js (hunk 1 above)
Comment (HristoYonkov) [reply src/controllers/eventController.js:169]: Yes, with this function i parse the year and the mont into date and then i compare the dates in the DB and then I take em.

**Review by Getsov — APPROVED** [review]

**Pull request merged**